tests/integration/GlobalFunctionsTest.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace GroupLife\Core\tests\integration;
+
+use PHPUnit\Framework\TestCase;
+
+class GlobalFunctionsTest extends TestCase
+{
+    public function testGetMigrationsPath()
+    {
+        self::assertFileExists(getMigrationsPath() . '/Version20210913225128.php');
```

Comment:
Here you need a directory separator too. Or move ending slash to `getMigrationsPath()` method.